### PR TITLE
Create entities

### PR DESCRIPTION
**Pull request by maximKalinov**

### CODE REVIEW

File: AreaMonitor.WebApi/AreaMonitor.WebApi.Domain/Mask.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace AreaMonitor.WebApi.Domain
+{
+    public class Mask
+    {
+        public int MaskId { get; set; }
+        public string Pattern { get; set; }
+        public string OriginCode { get; set; }
```
Comment (AntsiferauGodel) [inline AreaMonitor.WebApi/AreaMonitor.WebApi.Domain/Mask.cs]: If you use Current, why don't use Original? what do you think?

Comment (pipalalala) [inline AreaMonitor.WebApi/AreaMonitor.WebApi.Domain/Mask.cs]: Please, rename this to OriginalCode

Comment (maximKalinov) [reply AreaMonitor.WebApi/AreaMonitor.WebApi.Domain/Mask.cs]: Done

Comment (maximKalinov) [reply AreaMonitor.WebApi/AreaMonitor.WebApi.Domain/Mask.cs]: Done

**Pull request merged**